### ISSUE TITLE
Helpers.routeAndCall throws Exception [TestCaseProvided]

### ISSUE THREAD

**Issue by eximius313** [playframework/playframework#8177]
If you call `Helpers.routeAndCall`  an exception is thrown:
```
[error] Test controllers.HomeControllerTest.shouldCallServiceByRouteAndCall failed: java.lang.RuntimeException: java.lang.ClassNotFoundException: Routes, took 0.212 sec
[error]     at play.test.Helpers.routeAndCall(Helpers.java:387)
[error]     at controllers.HomeControllerTest.shouldCallServiceByRouteAndCall(HomeControllerTest.java:61)
[error]     ...
[error] Caused by: java.lang.ClassNotFoundException: Routes
[error]     at java.net.URLClassLoader.findClass(Unknown Source)
[error]     at java.lang.ClassLoader.loadClass(Unknown Source)
[error]     at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
[error]     at java.lang.ClassLoader.loadClass(Unknown Source)
[error]     at play.test.Helpers.routeAndCall(Helpers.java:383)
[error]     ... 37 more
```
Test case: [PlayRouteTest.zip](https://github.com/playframework/playframework/files/1653247/PlayRouteTest.zip)

**Comment by kujosHeist** [comment]
any work around for this?

**Comment by mkurz** [comment]
`routeAndCall` should just be used when using the routing DSL. Otherwise just use the `route(...)` method. I will remove the outdated `routeAndCall` methods: #10907

**Comment by mkurz** [comment]
Fixed by #10907
